src/lib/api.ts

```diff
@@ -8,7 +8,8 @@ import remarkGfm from 'remark-gfm';
 import remarkSlug from 'remark-slug';
 import type { Pluggable } from 'unified';
 import { codeExamples } from '~/docs/fuel-graphql-docs/src/lib/code-examples';
-import { codeImport } from '~/docs/fuels-wallet/packages/docs/src/lib/code-import';
+import { codeImport as walletCodeImport } from '~/docs/fuels-wallet/packages/docs/src/lib/code-import';
+import { codeImport } from './plugins/code-import';
```

Comment:
why do we need two different `code-imports` right here?

Reply:
We need to have more flexibility in the code import for the guides, so I think it's best to decouple that with the wallet. Pulling the wallet one in from the submodule ensures it won't break.

Reply:
yup, but we have a bunch of similar code on both files, we definitively could have just one file and trait this different cases inside of it, maybe if you create a class separating concerns and think better about the design of this class you can achieve something good